docs/faq.rst

```diff
@@ -219,6 +219,39 @@ Others
     .. code-block:: shell
 
         pip install --upgrade yasa
+        pip install --upgrade yasa
+    
+
+.. ----------------------------- RELATED PROJECTS -----------------------------
+.. raw:: html
+
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h5 class="panel-title">
+          <a data-toggle="collapse" href="#related_projects">Are there any related projects?</a>
+        </h5>
+      </div>
+      <div id="related_projects" class="panel-collapse collapse">
+        <div class="panel-body">
+
+**YASA Flaskified**
```

Comment:
LGTM but I'd love to hear from @remrama's before final approval.

Reply:
- The docs just got changed to restructured text (instead of HTML). The formatting is much simpler, but I'm not sure how to handle this merge (since #194 is already merged into `master`). If anyone needs help with this, let me know.
- Did an extra `pip install --upgrade yasa` get added to Line 222? If so, that should be removed.
- I think the dropdown heading could be more informative than **Are there any related projects?** What about **Can I use YASA online?** or **Can YASA be deployed on a remote server?** If that makes sense...